dashmips/syscalls/print_syscalls.py

```diff
@@ -2,41 +2,58 @@
 from . import mips_syscall
 from ..models import MipsProgram
 from ..utils import intify
+from struct import Struct
 
 
 @mips_syscall(4)
 def print_string(program: MipsProgram):
-    """Print string at address provided in $a0."""
+    """Print string.                 $a0 = address of null-terminated string to print."""
     address = program.registers["$a0"]
     string = program.memory.read_str(address)
-    print(string, end="")
+    print(string, end=" ")
 
 
 @mips_syscall(11)
 def print_char(program: MipsProgram):
-    """Print string at address provided in $a0."""
+    """Print character.              $a0 = character to print."""
     character = chr(program.registers["$a0"] & 0xFF)
     if ord(" ") <= ord(character) <= ord("~") or character in ("\n", "\t", "\r"):
         # is printable
-        print(character, end="")
+        print(character, end=" ")
 
 
 @mips_syscall(1)
 def print_int(program: MipsProgram):
-    """Print Int in Dec."""
-    print(program.registers["$a0"], end="")
+    """Print Int in Dec.             $a0 = integer to print."""
+    print(program.registers["$a0"], end=" ")
 
 
 @mips_syscall(34)
 def print_hex(program: MipsProgram):
-    """Print Int in Hex."""
-    print(f"{program.registers['$a0'] & 0xFFFFFFFF:08x}", end="")
+    """Print Int in Hex.             $a0 = integer to print."""
```

Comment:
Sorry to make you go over these docs again can we instead of having to maintain consistent spacing in each docstring maybe add to the docs_main code that will break on the first period and make the nicely formatted table.